Scoop/Controller/Profile/EditProfileViewController.swift

```diff
@@ -561,8 +567,95 @@ class EditProfileViewController: UIViewController {
             make.height.equalTo(textFieldHeight)
         }
     }
+
+    func setupDeleteButton() {
+        deleteButton.setTitle("Delete account", for: .normal)
+        deleteButton.setTitleColor(UIColor.white, for: .normal)
+        deleteButton.titleLabel?.font = UIFont.bodyBold
+        deleteButton.backgroundColor = UIColor.notification
+        deleteButton.layer.cornerRadius = 25
+        mainStackView.addArrangedSubview(deleteButton)
+
+        deleteButton.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(40)
+            make.height.equalTo(51)
+        }
+    }
+    
+    private func setupGradientView() {
+        gradientLayer.colors = [
```

Comment:
Maybe add this to the UIColor Extension, in case we need to utilize the same type of gradient again